tutorials/README.md

```diff
@@ -0,0 +1,26 @@
+
+# RabbitMQ C AMQP tutorials
+
+## Introduction
+
+This directory holds example tutorial code that implement the six tutorials 
+found at: 
+
+ - <https://www.rabbitmq.com/getstarted.html>
+
+#### Prereqs:
+- PHP5.6.x
```

Comment:
We officially support PHP > 5.3 (and I think we just drop 5.3 from tests and at this time you can even run php-amqp even on 5.3).